module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -39,19 +39,29 @@ Read the `SingleCellExperiment` package documentation (and e-book) [**here**](ht
 | `SingleCellExperiment` | [`SingleCellExperiment()`](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html) | Single Cell Experiment| Creates a `SingleCellExperiment` object  |
 | `SingleCellExperiment`| [`colData()`](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#2_Creating_SingleCellExperiment_instances) | Column Data | Extracts and stores cell-level metadata that describes features of the `SingleCellExperiment` object|
 | `SingleCellExperiment`| [`rowData()`](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#2_Creating_SingleCellExperiment_instances)  | Row Data   | Extracts and stores gene-level metadata that describes features of the `SingleCellExperiment` object|
+| `SingleCellExperiment`| [`assays()` and `assay()`](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#2_Creating_SingleCellExperiment_instances)  | Assay(s) | Extracts or assigns a given assay(s) from a `SingleCellExperiment` object|
```

Comment:
Do we want to link to the `SummarizedExperiment`  documentation to explain this and assayNames?(with some kind of note that the same functions apply at the top)? https://rdrr.io/bioc/SummarizedExperiment/man/SummarizedExperiment-class.html

Reply:
> (with some kind of note that the same functions apply at the top)

Can you clarify ^?

Reply:
What I meant here was that we are linking to files defined with respect to `SummarizedExperiment` objects, and it may not be immediately clear that they also work on `SingleCellExperiment` objects. So I wondered if we might want a note that says "Some of the functions below are defined in `SummarizedExperiment`, but can be applied to `SingleCellExperiment because the latter is a special version of the former" (Not those words!)

Reply:
Ah you know, I didn't even notice that! Will do